app/services/conferences_calendar_service.rb

```diff
@@ -0,0 +1,57 @@
+class ConferencesCalendarService
```

Comment:
What do you think about make this service stateless and delegate the responsibility of returning a full/not-full to itself? Suggestion:

```ruby
# Create or Build (not enough context)
class CreateConferenceCalendarService
  def self.call(calendar, conference, is_full_calendar)
    new(calendar, conference, is_full_calendar).call
  end
  
  def initialize(calendar, conference, is_full_calendar)
    @calendar = calendar
    @conference = conference
    @is_full_calendar = is_full_calendar
  end
  private_class_method :new
  
  def call
    if full?
      ...
    else
      ...
    end
  end
  
  private
  
  def full?
    @is_full_calendar
  end
end
```